.github/workflows/build_frp.yml

```diff
@@ -62,12 +76,119 @@
             git config --global url."https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com".insteadOf "https://github.com"
             chmod +x setup
             ./setup -b
-            ls
-          
+    
+        - name: getting commit id for workflow_dispatch
+          uses: actions/github-script@v6.3.3
+          id: find_commit
+          if: ${{ startsWith(env.SOURCE_BRANCH,'rel-') || env.SOURCE_BRANCH == 'main' }}
+          with:
+            github-token: ${{ secrets.PIPELINES_REPO_TOKEN  }}
```

Comment:
We have 2 secret with same purpose. Use `CIPIPELINE_USER_PASSWORD` instead of `PIPELINES_REPO_TOKEN`;